scripts/install_staging.sh

```diff
@@ -0,0 +1,82 @@
+# Update your system packages
+sudo apt update 
+
+# Install additional required libraries and tools
+sudo apt install --assume-yes make build-essential git clang curl libssl-dev llvm libudev-dev protobuf-compiler
+
+# Install rust and cargo
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+
+# Update your shell's source to include Cargo's path
+source "$HOME/.cargo/env"
+
+# Clone subtensor and enter the directory
+cd ../
+git clone https://github.com/opentensor/subtensor.git
```

Comment:
maybe check if subtensor already exists. also make sure that when the demo ends the subtensor setup is back to its original state